tests/functional/lti_certification/test_lti_certification_1_1.py

```diff
@@ -16,15 +18,12 @@ class TestLTICertification(TestBaseClass):
     OAUTH_NONCE = "38d6db30e395417659d068164ca95169"
     OAUTH_CLIENT = oauthlib.oauth1.Client(OAUTH_CONSUMER_KEY, SHARED_SECRET)
 
+    @pytest.mark.usefixtures("http_intercept")
     def test_a_good_request_loads_fine(self, app, lti_params_1x):
         result = self.lti_launch(app, lti_params_1x, status=200)
 
         self.assert_response_is_html(result)
 
-        # Check a random parameter to see it's passed to the body
-        assert "tool_consumer_instance_guid" in result.text
-        assert lti_params_1x["tool_consumer_instance_guid"] in result.text
```

Comment:
Some of the output doesn't do this, and frankly we don't care. 200 and HTML is good enough